source/_includes/common-tasks/filters.md

```diff
@@ -1,20 +1,25 @@
 Filters are applied as follows:
 
-1. No includes or excludes - pass all entities
+1. No includes or excludes - all entities
 2. Includes, no excludes - only include specified entities
 3. Excludes, no includes - only exclude specified entities
 4. Both includes and excludes:
    - Include domain and/or glob patterns specified
-      - If domain is included, and entity not excluded or match exclude glob pattern, accept
-      - If entity matches include glob pattern, and entity does not match any exclude criteria (domain, glob pattern or listed), accept
-      - If domain is not included, glob pattern does not match, and entity not included, reject
+      - If entity is included, accept
```

Comment:
Let's try to be more explicit, this refers to the entity being named directly, as opposed to being included via a domain or glob?